### ISSUE TITLE
Only lookup extensions locally?

### ISSUE THREAD

**Issue by hax** [barhun/extensionless#7]
It seems extensionless currently apply globally, lookup extensions even for the packages in the `node_modules`, in most cases we don't need that. Shall we only lookup locally and have a configuration to enable it for `node_modules` if really needed?

(Ideally I would like to enable lookup package by package, but it seems not easy to implement.)

**Comment by barhun** [comment]
Yes, it works globally but it tries to resolve package imports by adding extensions only in the case of a failure with the provided module (deep) specifier.

See: #2

**Comment by barhun** [comment]
BTW, since module resolvers/loaders work as a chain, this one has to call the next one at some point if we don't want to handle package imports ourselves. Only when those loader in the chain all fail to resolve the package import, this one tries with added extensions; so not much has changed with that one.
